### PR TITLE
feat: ui enhancements for handling the application through ahjo (HL-1167)

### PR DESCRIPTION
**Pull request by sirtawast**
## Description :sparkles:

* Add fancy editor for decision text styling
* Add prompts to prevent accidental submission
* "Success" message after submission
* Change LoadingSpinners to LoadingSkeletons here and there
* Write browser test to go through the new process
* Split components to smaller pieces

### CODE REVIEW

**Comment by EmiliaMakelaVincit** [comment]
Could you rebase this onto the current HL-1166 branch? This is currently based on an older revision of that, and the "files changed" view includes all the changes made there, making it tenuous to figure out what changes were made in this one specifically.

**Review by EmiliaMakelaVincit — COMMENTED** [review]
Frontend reviewed and backend parts also skimmed. There were a few things I wanted to point out but largely seems fine.

I also tried to briefly test the UI locally but couldn't get past the first step. For some reason, the application ID gets lost somewhere in the backend, though I see it's present in the payload.

```
benefit-backend          | Internal Server Error: /v1/decision-proposal-drafts/
benefit-backend          | Traceback (most recent call last):
benefit-backend          |   File "/usr/local/lib/python3.9/site-packages/django/db/backends/utils.py", line 84, in _execute
benefit-backend          |     return self.cursor.execute(sql, params)
benefit-backend          | psycopg2.errors.NotNullViolation: null value in column "application_id" violates not-null constraint
benefit-backend          | DETAIL:  Failing row contains (6, 2024-04-11 10:34:07.881378+00, 2024-04-11 10:34:07.881395+00, 2, accepted, , f, null, null, null, null).
benefit-backend          | 
benefit-backend          | 
benefit-backend          | The above exception was the direct cause of the following exception:
benefit-backend          | 
benefit-backend          | Traceback (most recent call last):
benefit-backend          |   File "/usr/local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 47, in inner
benefit-backend          |     response = get_response(request)
benefit-backend          |   File "/usr/local/lib/python3.9/site-packages/django/core/handlers/base.py", line 181, in _get_response
benefit-backend          |     response = wrapped_callback(request, *callback_args, **callback_kwargs)
benefit-backend          |   File "/usr/local/lib/python3.9/site-packages/sentry_sdk/integrations/django/views.py", line 85, in sentry_wrapped_callback
benefit-backend          |     return callback(request, *args, **kwargs)
benefit-backend          |   File "/usr/local/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
benefit-backend          |     return view_func(*args, **kwargs)
benefit-backend          |   File "/usr/local/lib/python3.9/site-packages/django/views/generic/base.py", line 70, in view
benefit-backend          |     return self.dispatch(request, *args, **kwargs)
benefit-backend          |   File "/usr/local/lib/python3.9/site-packages/rest_framework/views.py", line 509, in dispatch
benefit-backend          |     response = self.handle_exception(exc)
benefit-backend          |   File "/usr/local/lib/python3.9/site-packages/rest_framework/views.py", line 469, in handle_exception
benefit-backend          |     self.raise_uncaught_exception(exc)
benefit-backend          |   File "/usr/local/lib/python3.9/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
benefit-backend          |     raise exc
benefit-backend          |   File "/usr/local/lib/python3.9/site-packages/rest_framework/views.py", line 506, in dispatch
benefit-backend          |     response = handler(request, *args, **kwargs)
benefit-backend          |   File "/app/applications/api/v1/ahjo_decision_views.py", line 172, in patch
benefit-backend          |     proposal.save()
benefit-local-proxy      | 172.18.0.1 - - [11/Apr/2024:10:34:08 +0000] "PATCH /v1/decision-proposal-drafts/ HTTP/1.1" 500 226606 "https://localhost:3100/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36"
```

File: frontend/benefit/handler/src/components/applicationReview/useApplicationReview.ts [hunk 1]
```diff
@@ -141,8 +141,14 @@ const useApplicationReview = (): ExtendedComponentProps => {
           applicationNumber: application.applicationNumber,
         })
       );
+      window.history.replaceState(
```
[the remark this answers is not in the document]
Comment (sirtawast) [reply frontend/benefit/handler/src/components/applicationReview/useApplicationReview.ts:144]: This is to only clear the url argument. Using next's methods would do a rerender.

[the remark this answers is not in the document]
Comment (EmiliaMakelaVincit) [reply frontend/benefit/handler/src/components/applicationReview/useApplicationReview.ts:144]: Alright, fair enough.

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://kesaseteli-pr2907.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://tet-pr2907.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**EMPLOYER branch is deployed to platta: https://kesaseteli-employer-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**ADMIN branch is deployed to platta: https://tet-admin-ui-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tet-admin-ui-pr2907.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-employer-pr2907.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-handler-pr2907.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**YOUTH branch is deployed to platta: https://tet-youth-ui-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tet-youth-ui-pr2907.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://kesaseteli-pr2907.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://tet-pr2907.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://kesaseteli-pr2907.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://tet-pr2907.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://kesaseteli-pr2907.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://tet-pr2907.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**EMPLOYER branch is deployed to platta: https://kesaseteli-employer-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://helsinkilisa-ui-handler-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**APPLICANT branch is deployed to platta: https://helsinkilisa-ui-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-employer-pr2907.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://helsinkilisa-ui-pr2907.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe handler result is __failed__ for https://helsinkilisa-ui-handler-pr2907.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://kesaseteli-handler-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-handler-pr2907.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**YOUTH branch is deployed to platta: https://tet-youth-ui-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tet-youth-ui-pr2907.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**YOUTH branch is deployed to platta: https://kesaseteli-youth-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-youth-pr2907.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://kesaseteli-pr2907.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://tet-pr2907.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**EMPLOYER branch is deployed to platta: https://kesaseteli-employer-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**ADMIN branch is deployed to platta: https://tet-admin-ui-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-employer-pr2907.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://helsinkilisa-pr2907.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**APPLICANT branch is deployed to platta: https://helsinkilisa-ui-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://helsinkilisa-ui-handler-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tet-admin-ui-pr2907.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://helsinkilisa-ui-pr2907.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe handler result is __failed__ for https://helsinkilisa-ui-handler-pr2907.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://kesaseteli-handler-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-handler-pr2907.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**YOUTH branch is deployed to platta: https://tet-youth-ui-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**YOUTH branch is deployed to platta: https://kesaseteli-youth-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**EMPLOYER branch is deployed to platta: https://kesaseteli-employer-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://helsinkilisa-pr2907.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**ADMIN branch is deployed to platta: https://tet-admin-ui-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-employer-pr2907.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tet-admin-ui-pr2907.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://kesaseteli-handler-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-handler-pr2907.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tet-youth-ui-pr2907.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**YOUTH branch is deployed to platta: https://kesaseteli-youth-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-youth-pr2907.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://kesaseteli-pr2907.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://tet-pr2907.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**EMPLOYER branch is deployed to platta: https://kesaseteli-employer-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**ADMIN branch is deployed to platta: https://tet-admin-ui-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://helsinkilisa-pr2907.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://helsinkilisa-ui-handler-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**APPLICANT branch is deployed to platta: https://helsinkilisa-ui-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tet-admin-ui-pr2907.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-employer-pr2907.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://helsinkilisa-ui-pr2907.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe handler result is __failed__ for https://helsinkilisa-ui-handler-pr2907.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://kesaseteli-handler-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-handler-pr2907.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**YOUTH branch is deployed to platta: https://tet-youth-ui-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tet-youth-ui-pr2907.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**YOUTH branch is deployed to platta: https://kesaseteli-youth-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-youth-pr2907.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://helsinkilisa-ui-handler-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://tet-pr2907.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://kesaseteli-pr2907.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**ADMIN branch is deployed to platta: https://tet-admin-ui-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**EMPLOYER branch is deployed to platta: https://kesaseteli-employer-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tet-admin-ui-pr2907.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://helsinkilisa-pr2907.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://helsinkilisa-ui-handler-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**APPLICANT branch is deployed to platta: https://helsinkilisa-ui-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe employer result is __failed__ for https://kesaseteli-employer-pr2907.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://helsinkilisa-ui-pr2907.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**YOUTH branch is deployed to platta: https://tet-youth-ui-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://kesaseteli-pr2907.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://tet-pr2907.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://kesaseteli-pr2907.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://tet-pr2907.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://tet-pr2907.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**EMPLOYER branch is deployed to platta: https://kesaseteli-employer-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://helsinkilisa-pr2907.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://helsinkilisa-ui-handler-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**APPLICANT branch is deployed to platta: https://helsinkilisa-ui-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**ADMIN branch is deployed to platta: https://tet-admin-ui-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-employer-pr2907.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://helsinkilisa-ui-pr2907.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe handler result is __failed__ for https://helsinkilisa-ui-handler-pr2907.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://kesaseteli-handler-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-handler-pr2907.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**YOUTH branch is deployed to platta: https://tet-youth-ui-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tet-youth-ui-pr2907.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**YOUTH branch is deployed to platta: https://kesaseteli-youth-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-youth-pr2907.dev.hel.ninja :laughing::tada::tada::tada:

**Review by EmiliaMakelaVincit — APPROVED** [review]
I think all code issues raised earlier have been fixed, so nothing left to say on that front.

As for the hands-on test, even with this new version, the new flow doesn't work with old applications, but by testing with a newly created one, it seems fine. Might still be worth checking if anything can be done to that.

**Comment by terovirtanen** [comment]
**APPLICANT branch is deployed to platta: https://helsinkilisa-ui-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://helsinkilisa-ui-handler-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://helsinkilisa-ui-pr2907.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://helsinkilisa-ui-handler-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe handler result is __failed__ for https://helsinkilisa-ui-handler-pr2907.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://kesaseteli-pr2907.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://tet-pr2907.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**EMPLOYER branch is deployed to platta: https://kesaseteli-employer-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**ADMIN branch is deployed to platta: https://tet-admin-ui-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://helsinkilisa-ui-handler-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://helsinkilisa-pr2907.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**APPLICANT branch is deployed to platta: https://helsinkilisa-ui-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-employer-pr2907.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tet-admin-ui-pr2907.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://helsinkilisa-ui-pr2907.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://helsinkilisa-ui-handler-pr2907.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://kesaseteli-handler-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-handler-pr2907.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**YOUTH branch is deployed to platta: https://tet-youth-ui-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**YOUTH branch is deployed to platta: https://kesaseteli-youth-pr2907.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tet-youth-ui-pr2907.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-youth-pr2907.dev.hel.ninja :laughing::tada::tada::tada:

**Pull request merged**